modules/apps/site-initializer/site-initializer-liferay-marketplace/site-initializer-liferay-marketplace-test/src/testFunctional/tests/MP2PublisherDashboard.testcase

```diff
@@ -0,0 +1,149 @@
+@component-name = "portal-liferay-online"
+definition {
+
+	property osgi.modules.includes = "site-initializer-liferay-marketplace";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "Site Initializer Liferay Marketplace";
+
+	var testSiteName = "Marketplace Site";
+	var testSiteURL = "marketplace-site";
+
+	setUp {
+		task ("Set up instance and create a new Marketplace Site") {
+			MP2Configuration.marketplaceSetUp();
+		}
+
+		task ("Create a New Account and assign a user") {
+			JSONMP2.addAccount();
+
+			MP2Configuration.addUserAndSetAccount(
+				accountEntryName = "QA Development",
+				accountRole = "Account Administrator",
+				regularRole = "Account Administrator (Regular)",
+				roleTitle = "Site Owner",
+				userEmailAddress = "taqa@acme.com",
+				userFirstName = "Test Account",
+				userLastName = "QA",
+				userScreenName = "taqa");
+
+			MP2Configuration.addUserAndSetAccount(
+				accountEntryName = "QA Development",
+				accountRole = "App Editor",
+				regularRole = "Power User",
+				roleTitle = "Site Owner",
+				userEmailAddress = "testqa@editor.com",
+				userFirstName = "Test",
+				userLastName = "QA",
+				userScreenName = "testqa");
+		}
+
+		task ("Login with the created user and go to Marketplace") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "taqa@acme.com",
+				userLoginFullName = "Test Account QA");
+
+			MP2Navigator.gotoMP2();
+		}
+	}
+
+	tearDown {
+		MP2Configuration.marketplaceInitializersTearDown(doLogout = "true");
+	}
+
+	@description = "Story ID: LPS-162211 | Test Summary: Verify if the Publisher admin user is able to click on the invited user record to see its details"
```

Comment:
Luana, I'm suggestion to all QA to set a new pattern to TestDecription to be more easier to read.

1 - my first suggestion was clean the "Story ID: " given verione understand that LPS-XXXXX is a Story ID reference.
2 - remove the "Test Summary: "
3 - as I talked before in our testmap moment I`m proposing give a better read to our description
so far everyone agreed but I'm open to suggestions and critics.

my oppose is remove vague works like Verify or can.
in this case my initial suggestion is something like that:

"Publisher admin should be able to read invited user record details"

Feel free to change.

Reply:
For sure. This pattern we use, is a pattern adopted by some teams e.g. LXC and AC so we had chosen to follow it. But, we can change it. I will make the changes and upload it to PR. Thanks!